packages/dashmate/src/listr/tasks/stopNodeTaskFactory.js

```diff
@@ -20,7 +20,7 @@ function stopNodeTaskFactory(
     return new Listr([
       {
         title: 'Check node is running',
-        skip: (ctx) => ctx.isForceReset,
```

Comment:
You need to keep both I guess, or preferably rename `isForceReset` in other places to `isForce`